application/src/test/java/uk/nhs/hee/tis/revalidation/controller/RecommendationControllerTest.java

```diff
@@ -77,24 +82,25 @@ class RecommendationControllerTest {
   private String recommendationId = faker.number().digits(8);
   private String firstName = faker.name().firstName();
   private String lastName = faker.name().lastName();
-  private LocalDate submissionDate = LocalDate.now();
-  private LocalDate dateAdded = LocalDate.now();
+  private final LocalDate submissionDate = now();
+  private final LocalDate dateAdded = now();
+  private final LocalDate today = now();
```

Comment:
I suggest going through the class and replacing references to `now()` with `today` unless there is a clear reason that it is different.  Otherwise, you should just inline this in the 1 place where `today` is currently called.